packages/2018-neighborhood-development/src/components/StudentEnrollmentTrends/index.js

```diff
@@ -1,19 +1,111 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { css } from 'emotion';
 
-import { CivicStoryCard, Placeholder } from '@hackoregon/component-library';
+import { CivicStoryCard, Dropdown, StackedAreaChart } from '@hackoregon/component-library';
+import { ungroupBy } from '@hackoregon/component-library/src/utils/dataHelpers';
+import { year, titleCase } from '@hackoregon/component-library/src/utils/formatters';
+
+import { fetchSchoolList, fetchSchoolData, setSchool } from '../../state/student-enrollment-trends/actions';
+import {
+  isSchoolListPending,
+  catchSchoolListErrors,
+  getSchoolList,
+  isSchoolDataPending,
+  catchSchoolDataErrors,
+  getSchoolData,
+  getSelectedSchool,
+  getProcessedSchoolData,
+} from '../../state/student-enrollment-trends/selectors';
+
+const DEFAULT_SCHOOL = 'Buckman';
+const CHARTA_CATEGORIES = [
+  'white',
+  'asian',
+  'underrepresented',
+];
+const CHARTA_LABELS = [
+  'White',
+  'Asian',
+  'Historically Underrepresented',
+];
+const CHARTB_CATEGORIES = [
+  'black',
+  'hispanic',
+  'multi_ethnic',
+  'native',
+  'pacific',
+];
+const CHARTB_LABELS = [
+  'Black/African American',
+  'Hispanic/Latino',
+  'Multi-Ethnic',
+  'American Indian/Alaska Native',
+  'Native Hawaiian/Pacific Islander',
+];
+
+const formatForSelector = arr => arr.map(name => ({ value: name, label: name }));
+const formatForChartA = d => ungroupBy(d, CHARTA_CATEGORIES, CHARTA_LABELS);
+const formatForChartB = d => ungroupBy(d, CHARTB_CATEGORIES, CHARTB_LABELS);
```

Comment:
So another reason to use selectors is to get the benefits of memoization. Since you aren't doing the `ungroupBy` in a selector, this code runs on every render instead of only the renders where `processedSchoolData` changes.

I doubt it's causing perf issues with this small of a dataset, just something to keep in mind.

Reply:
will keep in mind for larger datasets